core-api-library/library/src/main/java/net/gini/android/core/api/Resource.kt

```diff
@@ -0,0 +1,10 @@
+package net.gini.android.core.api
+
+sealed class Resource<T>(
+    val data: T? = null,
+    val message: String? = null,
```

Comment:
Please move the subclass specific fields into the subclass and it's constructor.